File: hw/bonilla/U3/HW21TemplateMethod/TemplateMethodjs/controller/BeverageTest.js
const Tea = require('../model/Tea');
const Coffee = require('../model/Coffee');

async function main() {
  const tea = new Tea();
  const coffee = new Coffee();

  console.log("\nMaking tea ..");
  await tea.prepareRecipe();

  console.log("\nMaking coffee ...");
  await coffee.prepareRecipe();
}

main();

File: hw/caizapanta/u3/HW21Template/src/main/java/ec/edu/espe/cafe/view/BeverageTest.java
package ec.edu.espe.cafe.view;
import ec.edu.espe.cafe.model.Coffee;
import ec.edu.espe.cafe.model.Tea;

/**
 * 
 * @author Tammy Caizapanta
 */

public class BeverageTest {

    public static void main(String[] args) {
        Coffee coffee = new Coffee();
        Tea tea = new Tea();
        
        System.out.println("Making tea: \n");
        tea.prepareRecipe();
        System.out.println("Tea ready");
        
        System.out.println("\nMaking coffee: \n");
        coffee.prepareRecipe();
        System.out.println("Coffee ready");

    }
}
